packages/rke2-sriov/charts/templates/deployment.yaml

```diff
@@ -27,12 +27,13 @@ spec:
       labels:
         name: sriov-network-operator
     spec:
-      nodeSelector:
-        node-role.kubernetes.io/master: "true"
```

Comment:
It looks like the upstream chart intended for this deployment to only run on control-plane nodes, do we need to maintain that?

Reply:
yes thats why I removed the nodeselector since the master role label is deprecated and added another toleration for control-plane, so that it can be scheduled on either labels

Reply:
Right, you added a toleration so it **can** be scheduled there, but you didn't add a new nodeSelector for the control-plane role, so now it'll just run anywhere - which I don't think is correct.